app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreenViewModel.kt

```diff
@@ -149,7 +149,8 @@ class OtherUserProfileScreenViewModel @Inject constructor(
                 }
                 is IgnoreConnectionRequestUseCaseResult.Success -> {
                     state = state.copy(connectionStatus = ConnectionStatus.NotConnected)
-                    connectionOperationState = ConnectionOperationState.SuccessConnectionIgnoreRequest()
+                    connectionOperationState = ConnectionOperationState.SuccessConnectionIgnoreRequest(state.userName)
+                    navigationManager.navigateBack(mapOf())
```

Comment:
```suggestion
                    navigationManager.navigateBack()
```

I believe it's ok if we don't pass the args explicitly, since we have a default arg =)

Reply:
Also, not related to this, but there's an issue with the `MaxLineLength` rule on detekt 🔍

Reply:
I forgot to remove it because it's connected with my next PR 😅